serverless/tensorflow/faster_rcnn_inception_v2_coco/nuclio/function-gpu.yaml

```diff
@@ -114,7 +114,7 @@ spec:
         - kind: RUN
           value: ln -s faster_rcnn_inception_v2_coco_2018_01_28 faster_rcnn
         - kind: RUN
-          value: pip install pillow pyyaml
+          value: pip install pillow=9.5.0 pyyaml
```

Comment:
Explain?

(Also, I think you need a second `=` there.)

Reply:
If the runtime version of python is 3.9 installed version of Pillow is 10.3.0(last version). But in the `faster_rcnn` code used methods that are not supported on Pillow>10.

Reply:
I see. In that case, it would be clearer to write the requirement as `'pillow<10'`, and add a comment explaining what you just said.

Reply:
Fixed it